articles/product-lifecycle/product-release-stages.md

```diff
@@ -0,0 +1,71 @@
+
+During this stage, we work with participants to
+
+* confirm that functionality aligns with the intended goal
+* validate that the solution is usable in practice
+* elicit suggested improvement
+
+During this stage, our goal is to ensure that the eventual release provides the maximum value to our customers.
+
+When participating in an early access program, you should understand the following:
+
+* Functionality is under active development, so we do not yet support it for production use.
+* We expect breaking changes, but will do our best to communicate them.
+* The program provides a private preview available to only a limited number of select subscribers.
+* Your feedback may help shape the General Availability (GA) or a subsequent release.
+
+Early Access offerings are subject to our Beta Service Terms, which you can view at [Legal](https://auth0.com/legal).
+
+## Beta (Private or Public)
+
+Beta releases give subscribers time to explore and adopt new product capabilities while providing final feedback prior to a General Availability (GA) release. Functionality is code-complete, stable, useful in a variety of scenarios, and believed to meet or almost meet quality expectations for a GA release. Beta releases may be restricted to a select number of subscribers (private) or open to all subscribers (public).
+
+When participating in a beta release, you should understand the following:
+
+* Functionality is feature complete, but we do not yet support it for production use.
+* Breaking changes may occur, and we will do our best to communicate them.
+* Your feedback may help prioritize improvements and fixes in a subsequent release.
+
+Beta releases are subject to our Beta Service Terms, which you can view at [Legal](https://auth0.com/legal).
+
+## General Availability
+
+General Availability (GA) releases are fully functional and available to all subscribers (limited by pricing tier) for production use. If a new release replaces an existing feature, we provide a period of backward compatibility in accordance with our deprecation policy and inform customers so they have time to adopt the new release.
+
+## Deprecation
+
+Deprecated features are not supported for use by new subscribers, are not actively being enhanced, and are being only minimally maintained. <dfn data-key="tenant">Tenants</dfn> using the feature at the time of deprecation will continue to have access. 
+
+Deprecations begin with a <dfn data-key="breaking-change">breaking change</dfn> and end when the referenced feature or behavior moves into the End of Life product release stage. During Deprecation, customers should engage in a <dfn data-key="migration">migration</dfn> to move away from the deprecated feature or behavior. To learn more, see [Migration Process](/product-lifecycle/migration-process).
```

Comment:
Kind of a nitpick, but should we mention something here about how the breaking change would only be experienced as a breaking change at the End Of Life? If I read this as literally as possible it sounds like 1. we break functionality, 2. customers migrate to the new functionality. 

I'm probably overthinking this line, maybe no one else would read it that way... and I'm not sure how to better phrase this. May something like 
> Deprecation beings with the introduction of new behavior that would be experienced as a breaking change if no action is taken to migrate away before the End Of Life...
?

Anyway, just a thought.

Reply:
Oh actually... just reading the migrations-process page. That breaks it down bit more how I think about it in my head
> When we must introduce a breaking change, we first deprecate the affected feature or behavior and announce the Deprecation to our customers.

So 1. feature is deprecated 2. migration performed 3. breaking change mitigated 🥳

Reply:
Heroku app is rebuilding, but see what you think about my change. I believe it sidesteps the problem and keeps it simple, while leaving questions about the full process on the migration process page.